### PR TITLE
feat(kd-9142): extend translations for recaptcha copy

### PR DESCRIPTION
**Pull request by keepforever**
## Description of this change

> extend translations for recaptcha copy

## Why is this change being made?
- [x] Chore (non-functional changes)
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How was this tested? How can the reviewer verify your testing?

> local dev
## Related issues

> https://ketch-com.atlassian.net/browse/KD-9142

## Checklist

- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I have evaluated the security impact of this change, and [OWASP Secure Coding Practices](https://owasp.org/www-project-secure-coding-practices-quick-reference-guide/migrated_content#) have been observed.
- [x] I have informed stakeholders of my changes.

### CODE REVIEW

**Review by suhail-ansari — APPROVED** [review]

File: src/index.ts [hunk 1]
```diff
@@ -1473,6 +1473,7 @@ export interface Translations {
   we_are_unable_to_fulfill_your_request?: string
   at_this_time_due_to_suspected_bot_like_behavior_as_indicated_by_our?: string
   verification?: string
+  and?: string
```
Comment (sethyates) [inline src/index.ts:1476]: Typically single word translations like this are an anti-pattern because different languages conjoin things differently.

**Review by sethyates — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (keepforever) [reply src/index.ts:1476]: @sethyates , I'm after this, where TOS and PP are links, with `"and"`sandwiched in-between.  Not sure a way around that.

image

Comment (sethyates) [reply src/index.ts:1476]: You would have a translation that has those two template replacements in there.

Comment (keepforever) [reply src/index.ts:1476]: So, you're suggesting I have a single entry in the translations `.json`.   How would I know where to split the string in each and every language so i could wrap some substrings in `<Link/>` and others not?

I see you're off for the rest of today.  I'll ping you on Monday and you can show me your solution of how you'd do that in the code.

image

**Pull request merged**